### PR TITLE
Symfony 3 compatibility changes

### PR DESCRIPTION
**Pull request by garf**

### CODE REVIEW

**Comment by dmaicher** [comment]
Thanks :+1: :smile: 

Can you fix the tests? Added Travis CI config now :wink:

**Comment by garf** [comment]
Done :) Should be green now :)

File: Tests/NodeVisitor/NodeActivatorTest.php [hunk 1]
```diff
@@ -25,18 +25,16 @@ class NodeActivatorTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->request = $this
```
Comment (dmaicher) [inline Tests/NodeVisitor/NodeActivatorTest.php]: `$this->request` and `$this->activator` not needed anymore? Do we need the setup method at all then? :blush:

**Review by dmaicher — CHANGES_REQUESTED** [review]

File: Tests/NodeVisitor/NodeActivatorTest.php (hunk 1 above)
Comment (garf) [inline Tests/NodeVisitor/NodeActivatorTest.php]: True :) I was eager to fix it :)

**Pull request merged**